src/check/package-json.test.js

```diff
@@ -6,13 +6,10 @@ const pkg = require(`${process.cwd()}/package.json`);
 
 const coreScripts = [
   'build',
-  'build:js',
-  'build:test',
   'clean',
   'lint',
   'start',
-  'test',
-  'version'
+  'test'
```

Comment:
I'm good with `watch` becoming a required script, but I don't think version/prepublish scripts should be included even though the generator adds them.